packages/itwin/grouping-mapping-widget/src/widget/components/PropertyTableToolbar.tsx

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { SvgAdd, SvgRefresh } from "@itwin/itwinui-icons-react";
+import { Button, IconButton } from "@itwin/itwinui-react";
+import "./PropertyTableToolbar.scss";
+
+export interface PropertyTableToolbarProps {
+  propertyType: string;
+  onClickAddProperty?: () => void;
+  refreshProperties: () => void;
+  isLoading: boolean;
+}
+
+export const PropertyTableToolbar = ({
+  propertyType,
+  onClickAddProperty,
+  refreshProperties,
+  isLoading,
+}: PropertyTableToolbarProps) => (
+  <div className={`gmw-property-table-toolbar`}>
+    {onClickAddProperty &&
+      <Button
+        startIcon={<SvgAdd />}
+        styleType='high-visibility'
```

Comment:
Inconsistent use of quotes. You have all 3 supported quote types, when 1 would be enough. You use backticks for a string above, but you do not do any string interpolation there.

Reply:
Linter should have picked this up. Fixed it.